mix_common.exs

```diff
@@ -94,11 +94,12 @@ defmodule Antikythera.MixCommon do
     case Mix.env() do
       :dev -> [
         :mix,     # Suppress warning about Mix.Task behaviour
-        :eex,     # Only used during compilation, suppress warning about EEx.Engine behaviour
         :ex_unit, # Suppress warnings about calling ExUnit functions in Antikythera.Test.*
       ]
       _ -> []
-    end
+    end ++ [
+      :eex,       # Only used during compilation, suppress warning about EEx.Engine behaviour
```

Comment:
Since EEx is used by macro, a gear shouldn't depend on `:eex` at runtime.
After your fix, a gear depends on `:eex` at runtime.
> _build/prod//lib/gear_name/ebin/gear_name.app:                             ex_aws_ses,sweet_xml,dodai_client_elixir,eex]},

I think the 3rd option is better like `suppress warning about EEx.Engine behaviour` comment here.
>   3. In case you don't want to add a requirement to :eex, you may optionally skip this warning by adding [xref: [exclude: [EEx]]] to your "def project" in mix.exs